docs/reference/search/retriever.asciidoc

```diff
@@ -371,6 +374,118 @@ GET movies/_search
 ----
 // TEST[skip:uses ELSER]
 
+[[rescorer-retriever]]
+==== Rescorer Retriever
+
+The `rescorer` retriever re-scores only the results produced by its child retriever.
+For the `standard` and `knn` retrievers, the `window_size` parameter specifies the number of documents examined per shard.
+
+For compound retrievers like `rrf`, the `window_size` parameter defines the total number of documents examined globally.
+
+When using the `rescorer`, ensure its minimum rescore's `window_size` is:
```

Comment:
Maybe we could also mentioned what would happen if we misconfigure this? Would we get a parsing/validation exception, or maybe zero or unpredictable results?

Reply:
Reworded in https://github.com/elastic/elasticsearch/pull/118585/commits/9634945848e76093cbca87b14c8c25736009ff22